src/west/manifest.py

```diff
@@ -216,6 +219,18 @@ def _load(self, data, sections):
                                 format(name) +
                                 'be used to name a manifest project')
 
+            path = mp.get('path')
+            if path is not None:
```

Comment:
If not given, the implied path is the name of the project. What if I have a project named `west/subproject` or so? Do we still fail?

If so, would you mind adding some test coverage (another `invalid_xxx.yaml` should do it) to show that is true?